app/controllers/api/v1/labels.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'doorkeeper/grape/helpers'
+
+module API
+  module V1
+    class Labels < Grape::API
+      helpers Doorkeeper::Grape::Helpers
+
+      before { doorkeeper_authorize! }
+
+      before do
+        def current_account
+          Account.find(doorkeeper_token.resource_owner_id)
+        end
+      end
+
+      resource :labels do
+        desc 'List all labels for current account.'
+        get do
+          present current_account.labels, with: API::Entities::Label
+        end
+
+        desc 'Return a label by key.'
+        params do
+          requires :key, type: String, allow_blank: false, desc: 'Label key.'
+        end
+        route_param :key do
+          get do
+            label = Label.find_by(key: params[:key])
+            if label.present?
+              present label, with: API::Entities::Label
+            else
+              error!('Couldn\'t find Label', 404)
+            end
+          end
+        end
+
+        desc "Create a label with 'general' scope."
+        params do
+          requires :key, type: String, allow_blank: false, desc: 'Label key.'
+          requires :value, type: String, allow_blank: false, desc: 'Label value.'
+        end
+        post do
+          label =
+            current_account.labels.new(
+              key: params[:key],
+              value: params[:value],
+              scope: 'general'
```

Comment:
there is scope private or public no general scope.

Reply:
@mod @maksim-litvinov unfortunately public, private is reserved words, so I replaced them with synonyms general, restricted